### ISSUE TITLE
Flux Example - Declarative package with images from Registry1

### ISSUE THREAD

**Issue by RothAndrew** [defenseunicorns/zarf#169]
As a user of Zarf and Flux I want a Zarf package that simplifies the deployment of Flux, but keeps Registry1 as the source of the images that get run, so that I can have a simple way to deploy Flux without also needing to maintain a Utility Cluster.

Acceptance Criteria:
- New Zarf example `flux-from-iron-bank` (or some other appropriate name) exists
- It packages the manifests needed to deploy Flux, but not the images (they should just come from Iron Bank)

Nice-To-Have:
- The Flux manifests are pulled automatically from [Repo1](https://repo1.dso.mil/platform-one/big-bang/bigbang/-/tree/master/base/flux) so they don't need to be frequently updated

Notes:
- This example will be of limited usefulness until after the Native K8s Apply work is done. Once that is done we should be able to use it to deploy Flux to any cluster with ease.

**Comment by jeff-mccoy** [comment]
I don't really understand the value of this one.  If you're just using Zarf to install a manifest and not care about images at all, why not use kubectl?  Can we re-evaluate this one @RothAndrew?

**Comment by RothAndrew** [comment]
I think this is OBE now that #237 is merged. Closing for now
